modules/olm-multitenancy-solution.adoc

```diff
@@ -8,7 +8,14 @@
 
 While a *Multinamespace* install mode does exist, it is supported by very few Operators. As a middle ground solution between the standard *All namespaces* and *Single namespace* install modes, you can install multiple instances of the same Operator, one for each tenant, by using the following workflow:
 
+// In OSD/ROSA, dedicated-admins can create projects, but not namespaces.
+ifndef::openshift-dedicated,openshift-rosa[]
 . Create a namespace for the tenant Operator that is separate from the tenant's namespace.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+. Create a project (namespace) for the tenant Operator that is separate from the tenant's project (namespace).
+endif::openshift-dedicated,openshift-rosa[]
```

Comment:
This ^ is a bit confusing.
I like the [earlier wording](https://github.com/openshift/openshift-docs/pull/61793/files#diff-02d2daf3b27a9377b30f5f0776ba14fb5e28b45c60bc482cbfecf3e0af98e082R53) on how to create namespaces by first creating the project.